proxy.py

```diff
@@ -12,25 +12,26 @@ def __init__(self):
         super().__init__()
         self.logger = get_nio_logger("ConfigurationProxy")
 
-    def load_collection(self, name, url, token):
-        """ Retrieves a collection by its name
+    def load_configuration(self, url, token, org_id):
+        """ 
+        Retrieves an instance configruation by a instance_configuration_id 
+        and a instance_configuration_version_id
         """
-
-        self.logger.info("Loading '{}' collection".format(name))
-        payload = urlencode({
-            "collection": name
-        })
+        
         try:
             response = requests.get(
                 url,
-                params=payload,
-                headers=self._get_headers(token)
+                headers=self._get_headers(token, org_id)
             )
             if response.ok:
                 return response.json()
         except requests.exceptions.ConnectionError:
-            self.logger.exception("Failed to load collection")
+            self.logger.exception("Failed to load configuration")
 
     @staticmethod
-    def _get_headers(token):
-        return {"authorization": "bearer {}".format(token)}
+    def _get_headers(token, org_id):
+        return {
+            "authorization": "bearer {}".format(token), 
+            "nio-organization": org_id,
```

Comment:
`nio-organization` should be removed from here. @mattdodge with using `api_key` authentication should we have a different header than `authorization` that is specific for `api_key` authentication?

Reply:
Yeah, that probably makes sense. Using API keys as a bearer token when we already use JWTs there feels weird and would make it tough on the API to differentiate. I would either propose a custom auth type:
```
Authorization: apikey mykeygoeshereyoulikeit
```
or a different header like `nio-api-key`. I'm not sure which is more standard practice TBH

Reply:
If `Authorization: apikey XXX` is a valid authorization scheme I'd be for that 👍

Reply:
https://stackoverflow.com/questions/7802116/custom-http-authorization-header